include/onnxruntime/core/framework/float8.h

```diff
@@ -56,23 +56,24 @@ struct Float8E4M3FN {
       uint8_t e = static_cast<uint8_t>((b & 0x7F800000) >> 23);  // exponent
       uint32_t m = static_cast<uint32_t>(b & 0x007FFFFF);        // mantissa
       if (e != 0) {
-        if (e < 117) {         // 0b1110101
-        } else if (e < 118) {  // 0b1110110
-          val |= 1;
-          if ((m >> 23) & 1) {
-            // rounding
-            val += 1;
+        if (e < 117) {
```

Comment:
>       if (e < 117) { [](http://example.com/codeflow?start=1&length=21)

I think here it should be 116 instead of 117. if e == 116, the 2^-9 x 1.fraction >= 2^-9, which is the min subnorm.